pkgs/tools/inputmethods/interception-tools/space2meta.nix

```diff
@@ -0,0 +1,30 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  cmake,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "space2meta";
+  version = "0.2.0";
+
+  src = fetchFromGitLab {
+    group = "interception";
+    owner = "linux/plugins";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-MvpIe230I0TNzTO6ol1+DrpcFgqw0gF9Z22WMQLujb4=";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  meta = with lib; {
+    homepage = "https://gitlab.com/interception/linux/plugins/space2meta";
+    description = "Turn your space key into the meta key (a.k.a. win key, OS key, super key) when chorded to another key (on key release only).";
```

Comment:
The description should be one short factual sentence, see: 
https://github.com/NixOS/nixpkgs/tree/master/pkgs#meta-attributes for more details

Reply:
Thanks, will change this tomorrow.

Just so you know where I got this line, 
I noticed the description in caps2esc came from their [readme](https://gitlab.com/interception/linux/plugins/caps2esc/-/blob/master/README.md?ref_type=heads#caps2esc)
so I followed that convention [here](https://gitlab.com/interception/linux/plugins/space2meta/-/blob/master/README.md?ref_type=heads#space2meta)